components/HowItWorks/HowItWorks.tsx

```diff
@@ -60,18 +61,15 @@ export function HowItWorks() {
   return (
     <OuterWrapper ref={howItWorksRef} id="how-it-works" style={style}>
       <InnerWrapper>
-        <HeaderWrapper>
-          <Title>How it works</Title>
-          <Header>The Optimistic Oracle verifies data in stages</Header>
-        </HeaderWrapper>
-        {steps.map(({ header, text, subText, animationData }, index) => (
+        <SectionHeader title="How it works" header="The Optimistic Oracle verifies data in stages" />
+        {steps.map(({ header, text, subText, animationData }, index, arr) => (
           <StepWrapper key={header}>
             <StepNumberWrapper>
-              <StepNumber>0{index + 1}</StepNumber>
-              <StepLine />
+              <StepNumber invert={index % 2 === 1}>0{index + 1}</StepNumber>
```

Comment:
this looks good but did we give up on the idea of animating the numbers / lines?